CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+
+## Ground Rules
+
+* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
+* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds.
+
+
+## Your First Contribution
+
+Unsure where to begin contributing to TJ-Bot? You can start by looking through these labels!
+* [good first issue](https://github.com/Together-Java/TJ-Bot/issues/?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
+* [help wanted](https://github.com/Together-Java/TJ-Bot/issues/?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)  - issues which should be a bit more involved than good first issues.
+
+Working on your first Pull Request? You can check these resources:
+* http://makeapullrequest.com/
+* http://www.firsttimersonly.com/
+
+
+At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first! :tada: 
+
+
+## Getting started
+
+1. Create your own fork of the repository.
+2. Run the tests. We only take pull requests with passing tests, and it's great to know that you have a clean slate.
+3. To comply with our style guidelines use `./gradlew spotlessApply`. PRs should not include commits with autostyle runs. Instead, fix any style violations before committing changes.
+4. Each pull request should be clear, concise and informative: Please consider checking these resources: [writing a great pull request](https://www.pullrequest.com/blog/writing-a-great-pull-request-description/) and [unwritten guide to pull requests](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
+5. Each pull request should implement one feature or bugfix. If you want to add or fix more than one thing, submit more than one pull request.
+6. Each commit name and message should be clear, concise and informative: Please consider checking these resources: [writing a commit message](https://chris.beams.io/posts/git-commit/) and [writing a good commit message](https://dev.to/chrissiemhrk/git-commit-message-5e21)
+7. Each commit should be small, contained and should solve only one problem.
+8. Pull request should not include any merge commits. To update the branch of your pull request use `git pull --rebase`.
+
+
+## Community
+
+You can chat with the TJ-Bot users and devs in our [discord server](https://discord.gg/UfasCQNv)!
+
+#### If you find a security vulnerability, do NOT open an issue. Instead, consult the instructions in the [Security Policy](https://github.com/Together-Java/TJ-Bot/blob/master/SECURITY.md)
```

Comment:
```suggestion
#### If you find a security vulnerability, do NOT open an issue. Instead, consult the instructions in the [Make in progress](https://github.com/Together-Java/TJ-Bot/blob/master/SECURITY.md)
```
Rename it so it's clear till it's added?

Reply:
I would just leave it as it is and create the file later. Its just a red-link until then, which is fine.

Reply:
If it's displayed as a red-link sure